### ISSUE TITLE
question for fine-tune model on another dataset

### ISSUE THREAD

**Issue by lixingyi8947** [facebookresearch/maskrcnn-benchmark#445]
## ❓ Questions and Help
May I ask that when fine-tuning a pretrained model (say faster R-CNN) on another dataset, which metric should be monitoring most in order to tune the learning rate. Is that total loss or other individual loss? Will the /tools/train_net.py output a loss curve? Thanks a lot!

**Comment by fmassa** [comment]
The total loss is a good proxy, but overall what you ultimately want is the mAP on the validation set.

`tools/train_net.py` outputs the losses in a text file, you'd need to parse it or adapt the code to plot straight away to TensorBoard / Visdom / etc.

`tools/train_net.py` will store model checkpoints every x iterations, you can evaluate the quality of the checkpoint on the validation set using `tools/test_net.py`.
